central/complianceoperator/v2/report/manager/format/formatter.go

```diff
@@ -8,11 +8,14 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/central/complianceoperator/v2/report"
+	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/csv"
 )
 
 const (
-	EmptyValue = "Data not found for the cluster"
+	EmptyValue           = "Data not found for the cluster"
```

Comment:
nitpick: it looks like this constant is used only locally:
```suggestion
	emptyValue           = "Data not found for the cluster"
```

Reply:
Done